api/apps/articles/model.coffee

```diff
@@ -385,9 +387,40 @@ typecastIds = (article) ->
     biography_for_artist_id: ObjectId(article.biography_for_artist_id) if article.biography_for_artist_id
     super_article: if article.super_article?.related_articles then _.extend article.super_article, related_articles: article.super_article.related_articles.map(ObjectId) else {}
 
+sendArticleToSailthru = (article, cb) =>
+  images = {}
+  tags = article.keywords.concat ['article']
+  tags = tags.concat ['artsy-editorial'] if article.author_id is "503f86e462d56000020002cc"
+  tags = tags.concat ['magazine'] if article.featured is true
+  if article.email_metadata?.image_url
+    images =
+      full: url: crop(article.email_metadata.image_url, { width: 1200, height: 706 } )
+      thumb: url: crop(article.email_metadata.image_url, { width: 900, height: 530 } )
+  if NODE_ENV is 'production'
+    response = sailthru.apiPost 'content',
+      url: "#{FORCE_URL}/article/#{article.slug}"
+      date: article.published_at
+      title: article.email_metadata?.headline or article.thumbnail_title
+      author: article.email_metadata?.author or article.author?.name
+      tags: tags
+      images: images
+      spider: 0
+      vars:
+        credit_line: article.email_metadata?.credit_line
+        credit_url: article.email_metadata?.credit_url
+    , (err, response) =>
+      return cb(err) if err
+      cb(null)
+
 sanitizeAndSave = (callback) -> (err, article) ->
   return callback err if err
-  db.articles.save sanitize(typecastIds article), callback
+  # Send new content call to Sailthru on any published article save
+  if article.published
+    sendArticleToSailthru article, (err) =>
+      console.log 'Log an error' if err
```

Comment:
We actually have a `onPublish` callback already and I think this [belongs there](https://github.com/artsy/positron/blob/master/api/apps/articles/model.coffee#L284). That would allow you to leverage the [`if article.published`](https://github.com/artsy/positron/blob/master/api/apps/articles/model.coffee#L251) logic that already exists.